### ISSUE TITLE
[ERROR] Database file is locked

### ISSUE THREAD

**Issue by ilfarpro** [MrTigreroux/TigerReports#117]
TigerReports version 5.0.16
Purpur 1.18.2-1628 build
```
[ср 16:08:24 ERROR Minecraft] [TigerReports] An error has occurred with the database:
org.sqlite.SQLiteException: [SQLITE_BUSY]  The database file is locked (database is locked)
    at org.sqlite.core.DB.newSQLException(DB.java:1030)
    at org.sqlite.core.DB.newSQLException(DB.java:1042)
    at org.sqlite.core.DB.execute(DB.java:881)
    at org.sqlite.core.DB.executeUpdate(DB.java:922)
    at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:98)
    at TigerReports.jar//fr.mrtigreroux.tigerreports.data.database.Database.update(Database.java:59)
    at TigerReports.jar//fr.mrtigreroux.tigerreports.data.database.Database$1.run(Database.java:81)
    at org.bukkit.craftbukkit.v1_18_R2.scheduler.CraftTask.run(CraftTask.java:101)
    at org.bukkit.craftbukkit.v1_18_R2.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
    at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
    at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.base/java.lang.Thread.run(Thread.java:833)
```

**Comment by MrTigreroux** [comment]
Hello,

Could you give me the full logs from "[TigerReports] Enabling ..." to that error (removing sensitive informations like IPs..., you can send them in SpigotMC if you are not sure).
Thank you in advance.

**Comment by ilfarpro** [comment]
I can't give full logs right now, because logs with this error are too long (weeks) and fragmented to parts. I've restarted my server and once I catch this error again I will provide logs. Worth to mention that I previously tried to restart server and it did not helped to get rid of this error.
This error not breaks plugin, it continues to work as intended. I tried to use /report and /reports command and it's not triggering this error.
No any erorrs or warnings at plugin startup.
Please, do not close this issue for a while.

**Comment by MrTigreroux** [comment]
Ok, then could you give me one logs file (the last before that error) ?


> Worth to mention that I previously tried to restart server and it did not helped to get rid of this error.

Do you mean that your restarted your server and the error still hapenned ? If so it would be really interesting to see the logs (that I understand are really short) before the error happen (again).

**Comment by MrTigreroux** [comment]
Hello, the issue you had should be fixed in the new [version 5.1](https://www.spigotmc.org/resources/tigerreports.25773/update?update=475147). It took a lot of time but I think that I finally found the cause of the problem. Please let me know if it's not the case.
